src/frontend/App.tsx

```diff
@@ -65,6 +68,20 @@ function App() {
               element={<Navigate replace to="/hyperplaystore" />}
             />
             <Route path="/library" element={<Library />} />
+            <Route
+              path="/achievements"
+              element={
+                <AchievementsLayout>
+                  <Outlet />
+                </AchievementsLayout>
+              }
+            >
+              <Route index element={<Achievements />} />
+              <Route
+                path="/achievements/:id"
+                element={<GameAchievementDetails />}
+              />
+            </Route>
```

Comment:
Future thing, but maybe later we can move the Route's into a dedicated component and we initialize it here?

Reply:
created an issue for it here https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/issues/680

It is out of the scope of this pr so I will leave as is for now